templates/html/html--select-language.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Theme override for the basic structure of a single Drupal page.
+ *
+ * @see ./core/themes/stable/templates/layout/html.html.twig
+ */
+#}
+<!DOCTYPE html>
+<html{{ html_attributes }}>
+  <head>
+    <head-placeholder token="{{ placeholder_token }}">
+    <title>{{ head_title|safe_join(' | ') }}</title>
+    <css-placeholder token="{{ placeholder_token }}">
+    <js-placeholder token="{{ placeholder_token }}">
+  </head>
+  <body{{ attributes.addClass('language-' ~ html_attributes.lang).addClass('ecl-typography').addClass('page-splash') }}>
```

Comment:
We should rather use the `page-splash` via a preprocess so we don't need to override the html template.